### PR TITLE
Escape markdown characters when genereting a quote

### PR DESCRIPTION
**Pull request by marcofucci**
To avoid all possible XSS or phishing attacks, the user generated content is made markdown safe when generating a quote.

### CODE REVIEW

File: datahub/omis/quote/utils.py [hunk 1]
```diff
@@ -16,6 +18,23 @@
 QUOTE_TEMPLATE = PurePath(__file__).parent / 'templates/content.md'
 
 
+def escape_markdown(content):
```
Comment (elcct) [inline datahub/omis/quote/utils.py]: I would change the name to `escape_markdown_and_html`.

**Comment by marcofucci** [comment]
I agree and I tested with the current frontend.

File: datahub/omis/quote/utils.py (hunk 1 above)
Comment (marcofucci) [reply datahub/omis/quote/utils.py]: You can use html in markdown so I think escape markdown implies html as well, no?

Comment (elcct) [reply datahub/omis/quote/utils.py]: I see, that's true indeed. 
https://daringfireball.net/projects/markdown/syntax#html
Thanks

**Pull request merged**